packages/composite-checkout/src/components/checkout.js

```diff
@@ -154,13 +156,19 @@ const CheckoutWrapper = styled.div`
 
 function OrderSummaryStep( { OrderSummary } ) {
 	const localize = useLocalize();
+	const total = useTotal();
 
 	return (
 		<CheckoutStep
 			isActive={ false }
 			isComplete={ true }
 			stepNumber={ 0 }
-			title={ localize( 'You are all set to check out' ) }
+			title={
+				<CheckoutSummaryTitle>
+					<span>{ localize( 'You are all set to check out' ) }</span>
+					<CheckoutSummaryTotal>{ total.amount.displayValue }</CheckoutSummaryTotal>
```

Comment:
We might want to call `renderDisplayValueMarkdown()` on `displayValue` here just in case there's some markdown in it.

Reply:
Good idea. Will do!